homeassistant/components/plant.py

```diff
@@ -30,7 +31,9 @@
 READING_BRIGHTNESS = 'brightness'
 
 ATTR_PROBLEM = 'problem'
+ATTR_SENSORS = 'sensors'
 PROBLEM_NONE = 'none'
+ATTR_MAX_BRIGHTNESS_HISTORY = 'max. brightness in history'
```

Comment:
attributes should not have spaces or periods.

Reply:
ACK